docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4550,7 +4552,9 @@ The egress packets may be distinguished for further processing in the egress
 using the `instance` metadata. Note that a packet may not be both unicast and
 multicast; if the multicast group is set, it will override the unicast egress
 port. If the P4 `multicast_group` metadata is set to a value that is not
-programmed in the PRE, then the packet is dropped.
+programmed in the PRE, then the packet is dropped. Fallback ports for multicast
+uses one of the backup replicas when the primary multicast `port` becomes
+unavailable.
```

Comment:
that seems a bit too abstract for me. There are multiple "primary" multicast ports (in the example above: 5, 12, 18 and 24). So what happens when a single one fails? What happens when multiple ones fail (in particular if there are fewer backups that "primaries")?

Reply:
I updated the documentation so hopefully it is more concrete, but the idea is that each replica has an associated backup (a list of replicas). So when the primary port of the replica fails, the associated backup would choose one of it's replicas to replace the "primary" port of the old replica

Reply:
I think I agree with Antonin here... I would've expected that each replica has a list of backup ports basically, which it goes through in order as ports earlier in the list (and the primary port) fail. Are we sure that this list of fallback replicas can give the semantics we want? @smolkaj as well?

Reply:
From a few different perspectives, we want the replicas rather than just purely the fallback ports. 

In vanilla P4, the instance ID allows us to distinguish multiple packet copies in the egress pipeline. Because the P4Runtime standard requires that each (port, instance)-pair within a multicast group is unique, we need the instance ID as non-uniqueness would very likely indicate a bug (why would you want to send two identical packets out of the same port?)

For our use case, having the instance ID allows us to have 2 replicas at the same port but with different SMACs or VLANs

Reply:
As addressed in my other comment and offline discussion, we have replicas own the backup replicas. Closing this comment.